### DISCUSSION TITLE
i18n documentation

### DISCUSSION THREAD

**Discussion by CurryAyam** [quasarframework/quasar#12951] [Ideas / Proposals] [answered]
The quasar docs about App Internationalization should be update to the following 
```
export default createI18n({
  locale: 'en-US',
  globalInjection: true,
  messages
})
```

on the boot section!!

without the globalInjection:true, the $t will throw error 

![image](https://user-images.githubusercontent.com/88246252/160286662-486c3186-56f9-4531-bab2-629f5fd39dfc.png)
![image](https://user-images.githubusercontent.com/88246252/160286693-fedbf2c1-0e27-4380-a6bb-92639cbc0e88.png)

"vue-i18n": "^9.0.0",
"@intlify/vite-plugin-vue-i18n": "^3.3.1",
"@quasar/app-vite": "^1.0.0-alpha.0",

**Comment by rstoenescu** [comment, marked as answer]
Hi,

Thanks for reporting, BUT: `$t()` is discouraged for perf reasons. The Vue i18n docs clearly specify that the `v-t` directive should be used instead. https://vue-i18n.intlify.dev/guide/advanced/directive.html#t-vs-v-t
